conf/machine/nitrogen8m.conf

```diff
@@ -64,7 +64,7 @@ IMAGE_BOOT_FILES:append = " \
 
 # wic support
 WKS_FILE = "sdimage-bootpart.wks"
-WKS_FILE_DEPENDS:mx8 += " \
+WKS_FILE_DEPENDS:mx8m-generic-bsp += " \
```

Comment:
@GaryThomas please review this PR.

Reply:
@gibsson 👆🏼

Reply:
I am unsure it will apply or not. @Ossanes please check if dropping it, if it works.

Reply:
Ok. I'm testing.